packages/account-v2/src/constants/manualFormConstants.tsx

```diff
@@ -6,18 +6,23 @@ import NIMCSlipFront from '../assets/manual-upload/nimc-slip-front.svg';
 import PassportPlaceholder from '../assets/manual-upload/passport-placeholder.svg';
 import ProofOfAgeIcon from '../assets/manual-upload/proof-of-age.svg';
 
-export type TManualDocumentTypes = typeof MANUAL_DOCUMENT_TYPES[keyof typeof MANUAL_DOCUMENT_TYPES];
+export type TManualDocumentTypes = Exclude<
+    typeof MANUAL_DOCUMENT_TYPES[keyof typeof MANUAL_DOCUMENT_TYPES],
+    'birth_certificate'
+>;
 
 export const MANUAL_DOCUMENT_TYPES = Object.freeze({
+    birthCertificate: 'birth_certificate',
     drivingLicence: 'driving_licence',
     nationalIdentityCard: 'national_identity_card',
     nimcSlip: 'nimc_slip',
     passport: 'passport',
+    selfieWithID: 'selfie_with_id',
 });
 
 export const MANUAL_DOCUMENT_SELFIE = 'selfie_with_id';
```

Comment:
do we need to export it separately?
cant we use MANUAL_DOCUMENT_TYPES.selfieWithID  directly instead

Reply:
Will change this